tests/rptest/remote_scripts/stream_verifier_txn.py

```diff
@@ -0,0 +1,974 @@
+# Copyright 2024 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import argparse
+import confluent_kafka as ck
+import falcon
+import json
+import logging
+import signal
+import sys
+import threading
+
+from concurrent.futures import ThreadPoolExecutor
+from copy import deepcopy
+from dataclasses import dataclass
+from datetime import datetime
+from itertools import repeat
+from time import sleep
+from typing import Generator, IO, Tuple
+from wsgiref.simple_server import make_server
+
+global title
+app_name = "StreamVerifierTx"
+title = f"{app_name}, python transaction verifier worker"
+log_level = logging.INFO
+LOGGER_STARTUP = 'startup'
+LOGGER_MAIN = 'main'
+LOGGER_WEB_PRODUCE = "web_produce"
+LOGGER_WEB_CONSUME = "web_consume"
+CONSUMER_LOGGING_THRESHOLD = 1000
+
+
+def setup_logger(name: str = "", level: int = logging.DEBUG) -> logging.Logger:
+    """Create main loggeer or its child based on naming
+
+    Args:
+        name (str, optional): name of the logger. Defaults to "stream_verifier"
+        level (int, optional): Log level. Defaults to logging.DEBUG.
+
+    Returns:
+        logging.Logger: Logger created
+    """
+    handler = logging.StreamHandler(sys.stdout)
+    handler.setFormatter(
+        logging.Formatter(
+            '%(asctime)s - %(name)s - %(levelname)s - %(message)s'))
+    handler.setLevel(level)
+    _name = "stream_verifier"
+    if name is not None:
+        # Just simpel hierachy, no special handler
+        _name = f"{_name}.{name}"
+        logger = logging.getLogger(_name)
```

Comment:
To me, it is more convenient to use same method everywhere and get target logger by name with intuitive naming. Moreover getChild will require main logger too which is more complicated than just one func call. Also, these are the same as docs says
```
getChild(suffix)
Returns a logger which is a descendant to this logger, as determined by the suffix. Thus, logging.getLogger('abc').getChild('def.ghi') would return the same logger as would be returned by logging.getLogger('abc.def.ghi'). This is a convenience method, useful when the parent logger is named using e.g. __name__ rather than a literal string.
```